yarl/_url.py

```diff
@@ -1142,7 +1148,7 @@ def human_repr(self):
                     user,
                     password,
                     host,
-                    self._val.port,
+                    self.explicit_port,
```

Comment:
```suggestion
                    self._val.port,
```

Going to leave it as-is since CI is blocking merge otherwise